Automattic/wp-calypso
# Jetpack Backup: scroll back to credentials settings when expanding credentials

Resolves https://github.com/Automattic/jetpack-backup-team/issues/449

In Calypso Blue, if we are trying to edit server credentials, it goes to the top of the page. The idea is to prevent scrolling to the top. Here is a quick demo of current behavior:

https://github.com/Automattic/wp-calypso/assets/1488641/277cd659-92af-4917-a71f-d893074b79d1

## Proposed Changes
* Remove `setScroll` callback from Jetpack settings in Calypso Blue router definition. This will prevent Jetpack settings to scrolling up to the top when navigating on the credentials settings.

## Testing Instructions



* Spin up a Calypso live branch
* Pick a site with Jetpack VaultPress Backup and credentials configured
* Navigate to Settings > Jetpack (or `/settings/jetpack/site_slug`)
* Resize the window to make it a bit smaller
* Scroll down to `Remote server credentials settings`. It should look like this:
  ![CleanShot 2024-04-23 at 17 05 20@2x](https://github.com/Automattic/wp-calypso/assets/1488641/523f099e-78e6-4c30-827b-1164188cb19b)
* Click on the arrow to expand server credentials
* Ensure the browser does not scroll back to the top. It should behave similar to this video:

https://github.com/Automattic/wp-calypso/assets/1488641/5732b2df-04b9-4c4d-b37c-cee8486a0ad7

## Pre-merge Checklist
